### ISSUE TITLE
Deactivate comments on docs.marklogic.com

### ISSUE THREAD

**Issue by dmcassel** [marklogic/RunDMC#622]
Replace comments with links:

- report a bug (email docs team)
- ask a question (send to Stack Overflow)
- submit an example (email docs team)

**Comment by dmcassel** [comment]
Additional benefit of shutting off comments: #654.

**Comment by sammefford** [comment]
Hmm, I have always liked the *idea* of comments on our docs.  It makes me feel like we are open and community-connected because we allow the community to contribute directly on our docs.  As I've read the comments on our docs, the *reality* hasn't been as exciting as the idea.  I'm sure, however, that there are some places where the comments were really useful and more effective at delivering real-time corrections than a bug would be.  For now, my vote would be that we keep comments active but more actively curate them (as in, moderate down (hide) any comments that no longer provide value).  Does disqus allow us to moderate down (hide) comments that aren't helpful?

**Comment by dmcassel** [comment]
@sammefford I'm with you on the idea of comments, but the reality is that very few are well-placed and provide value to the community. More commonly, we see questions that would be better asked on Stack Overflow, or questions that aren't related to the page on which they are asked.

**Comment by sammefford** [comment]
Right.  Can we moderate those down so they're hidden by default and require a click to "see all comments"?  I've seen that a lot on popular forums like slashdot.

Also, any concern about losing the baby with the bath water?  Even if very few are well-placed, how much value is there in those few?

**Comment by dmcassel** [comment]
The moderation tools allow me to delete comments. I'm not sure we'd be able to selectively hide them, at least with the tools provided by disqus. 

My thought on the value: given that our community is relatively small, I'd rather concentrate what we have. A question asked on SO can be edited to improve formatting (and grammar, spelling, clarity), will be seen by more people, and good answers will be voted up. Questions in docs comments are seen by a handful of people; they are less likely to get any answer, let alone a good one. 

@kcoleman-marklogic, your thoughts?

**Comment by kcoleman-marklogic** [comment]
My bias is for removal, for the following reason:

I've been watching the comment traffic for about 6 months. In that period, receiving maybe 1-2 comments per week, I've yet to see one that was not a question that would be better asked on StackOverflow. SO reaches a wider community of experts, has better markup, and IMO has a better search trail in the wider world. (The questions are generally not germane to the documentation, either.)

When we originally added this capability to the app, StackOverflow was not A Thing in the way it is now, and ML had no presence there to speak of. I believe higher traffic on SO does more for our community image than an infinite number of comments in the docs.marklogic.com backwater.

Dave and I discussed the possibility of replacing the commenting capability with some sort of "Have a question?" link to drive more traffic in that direction.

I don't think it would be appropriate to moderate out someone's question. That's rude.

**Comment by sammefford** [comment]
I see a lot of comments on the docs by Jim Fuller.  They're not questions, but helpful tips.  There they kind of thing that works as a comment but wouldn't make sense on stackoverflow or in the official documentation.  I know I've seen many similar comments by other MarkLogic experts.  I understand the problem with questions being posted in the wrong place, or getting out of date, etc.  But I'd like to see if there are other options to address those issues.  It strikes me as overkill to remove comments completely.  Presumably, with comments traffic remaining low, we should be able to moderate them appropriately.

As far as rude, I'm not sure it's that simple.  StackOverflow is an example where poorly formed or inappropriate questions are quickly shut down.  I think the key is to provide some brief guidance on what's appropriate and what will get moderated down.
